dali/sasha/saarch.cpp

```diff
@@ -540,6 +542,11 @@ void WUiterate(ISashaCommand *cmd, const char *mask)
                         }
                     }
 
+                    if (wuCount < wuStart)
```

Comment:
I'm not convinced the wuStart logic here is right, or necessary.
Paging was achecived before by passing in the last workunit as a before or after in the request.

The count should always be the number you want to return (not start+count).